server/server.js

```diff
@@ -1,5 +1,5 @@
 const app = require('./app');
 
-app.listen(3000, () => {
```

Comment:
Why are there two different ports?

Reply:
Had to change one. This is the old port i was using before we agreed